src/services/helpers/service.js

```diff
@@ -1,9 +1,8 @@
 const request = require('request-promise-native');
 const { GeneralError, MethodNotAllowed } = require('@feathersjs/errors');
 const logger = require('../../logger');
-
+const { Configuration } = require('@schul-cloud/commons');
```

Comment:
Codacy found an issue: [`@schul-cloud/commons` import should occur before import of `../../logger`](https://app.codacy.com/manual/schul-cloud/schulcloud-server/pullRequest?prid=5994445)